.github/workflows/build.yml

```diff
@@ -19,6 +19,7 @@ jobs:
               -DMODPLUG=ON
               -DWAVPACK=ON
             cmake_generator: Unix Makefiles
+            cmake_build_type: RelWithDebInfo
```

Comment:
Considering we're using the same type for every OS, how about we move this to be an environment variable for the job rather than putting it in the matrix? You can add it next to the macOS code signing variables.

Reply:
I have made this intentional individual configure able. This allows to testing single targets without making the cache void for all others.

Reply:
okay